docs/05-_Navigation_Interception.md

```diff
@@ -22,7 +22,39 @@ Example:
 
 As the value of the `target` attribute, you might consider one of the following keywords that have a special meaning in HTML:
 
-- `_top` - load the URL into the current browser tab or window
-- `_blank` - load the URL into a new browser tab or window
+-   `_top` - load the URL into the current browser tab or window
+-   `_blank` - load the URL into a new browser tab or window
 
 You can find more information on the `target` attribute at [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a).
+
+### Intercepting within-app navigation
```

Comment:
Perhaps change the `How to disable interception` header on line 11 to `How to disable interception for an HTML link`

And then change the header `Intercepting within-app navigation` to `Cancellable events before and after navigation`